modules/serverless-build-events-kn.adoc

```diff
@@ -0,0 +1,75 @@
+[id="serverless-build-events-kn_{context}"]
+= Building events
+
+You can use the builder-like interface of the `kn event build` command to build an event. You can then send that event at a later time or use it in another context.
```

Comment:
Here in "... You can then send that event at a later time..." it would be nice to show an example that combines `kn event build` + `kn event send` in order to demonstrate to users how to send the event 'in a later time'.

Reply:
@jrangelramos Thanks Jefferson; good suggestion.
@cardil Is it possible to somehow directly "feed" a `kn event build`-generated event to `kn event send` ?

Reply:
That's unfortunately a planned feature. It's not possible to do it currently :sob:

Reply:
@cardil Thanks Chris, we'll add it later. :crossed_fingers: